uber/templates/registration/shifts.html

```diff
@@ -82,7 +84,7 @@
             {% csrf_token %}
             <input type="hidden" name="shift_id" value="{{ shift.id }}" />
             <td style="padding-right:10px"> <input type="submit" value="Unassign" /> </td>
-            </form>
+	    </form>
```

Comment:
Super-nitpicky formatting thing: can you fix the formatting by changing the tabs to spaces?  We use spaces instead of tabs everywhere in this codebase and I'd rather not start mixing tabs and spaces in the same file.